src/test/erc4626/ERC4626.t.sol

```diff
@@ -0,0 +1,132 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.10;
+
+import {Vm} from "../../../lib/forge-std/src/Vm.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {DefiBridgeProxy} from "./../../aztec/DefiBridgeProxy.sol";
+import {RollupProcessor} from "./../../aztec/RollupProcessor.sol";
+import {IERC4626} from "./../../interfaces/erc4626/IERC4626.sol";
+import {VaultBridge} from "./../../bridges/erc4626/VaultBridge.sol";
+import {AztecTypes} from "../../aztec/libraries/AztecTypes.sol";
+
+import "forge-std/Test.sol";
+import "forge-std/console.sol";
+
+//Tested at block 14886873 may not work at other blocks
+// forge test --fork-block-number 14886873  --match-contract ERC4626 --fork-url https://mainnet.infura.io/v3/9928b52099854248b3a096be07a6b23c
+contract ERC4626 is Test {
+    DefiBridgeProxy defiBridgeProxy;
+    RollupProcessor rollupProcessor;
+
+    VaultBridge vaultbridge;
+
+    IERC20 constant maple = IERC20(0x33349B282065b0284d756F0577FB39c158F935e6);
+    IERC20 constant fei = IERC20(0xc7283b66Eb1EB5FB86327f08e1B5816b0720212B);
+    IERC4626 constant tribeVault = IERC4626(0x4f93Df7Bc0421C9401fD3099cCE4AfE7678B0c63);
+
+    IERC4626 constant vault = IERC4626(0x4937A209D4cDbD3ecD48857277cfd4dA4D82914c);
+
+    function _aztecPreSetup() internal {
+        defiBridgeProxy = new DefiBridgeProxy();
+        rollupProcessor = new RollupProcessor(address(defiBridgeProxy));
```

Comment:
Please refactor this test so that [example test structure](https://github.com/AztecProtocol/aztec-connect-bridges/tree/master/src/test/bridges/example) is followed (separation to E2E and unit test file). Using DefiBridgeProxy and RollupProcessor directly in test has been deprecated.